_posts/2025-03-26-kubeflow-1.10-release.md

```diff
@@ -0,0 +1,268 @@
+enhancing flexibility in pipeline execution.This update allows users to define dynamic resource limits using
+parameterized values, enabling more adaptable and reusable pipeline definitions.
+
+### Support for Loop Parallelism
+
+Kubeflow Pipelines 2.4.1 introduces a new [Parallelism Limit for `ParallelFor` tasks](https://github.com/kubeflow/pipelines/issues/8718),
+giving users the ability to run massively parallel inference pipelines, with more control over parallel execution in
+their workflows. This feature allows users to specify the maximum number of parallel iterations, preventing resource
+overutilization and improving system stability.
+
+### Implement SubDAG Output Resolution
+
+Kubeflow 1.10 ensures that [pipelines using nested DAGs](https://github.com/kubeflow/pipelines/pull/11196) work
+correctly and reliably when treated as components. Outputs from deeply nested DAGs will now resolve properly, avoiding
+broken dependencies.
+
+## Model Registry
+
+Model Registry introduces a new user interface and enhanced model management capabilities.
+
+### Model Registry UI
+
+The new Kubeflow [Model Registry UI](https://www.kubeflow.org/docs/components/model-registry/getting-started/#using-the-model-registry-ui)
+provides a user-friendly web interface for managing machine learning models within the Kubeflow platform. It centralizes
+model metadata, version tracking, and artifact management, streamlining MLOps workflows.
+
+Key features include:
+
+* Easy model registration with custom metadata
+* Comprehensive model management with filtering and sorting
+* Archiving capabilities
+* Version control
+* Metadata editing
+
+![Model Registry UI](../images/2025-03-26-kubeflow-1.10-release/model-registry-ui.png)
+
+The UI interacts with the Model Registry's REST API, making it accessible to users of all technical backgrounds and
+enhancing collaboration across data science, ML engineering, and MLOps teams.
+
+To get started with the Model Registry UI, which is currently in Alpha, you can follow the instructions
+[here](https://www.kubeflow.org/docs/components/model-registry/installation/#installing-on-kubeflow-platform).
+
+### Custom Storage Initializer
+
+The Model Registry Custom Storage Initializer (CSI) is a custom implementation of the KServe ClusterStorageContainer.
+This feature allows users to utilize Model Registry metadata to download and deploy models efficiently. With the newest
+release of the Model Registry, it is now possible to install and use the Custom Storage Initializer (CSI).
+
+You can find detailed installation instructions and a small example in the "Getting Started" section of the Model
+Registry component on the Kubeflow website.
+
+For additional information and future developments towards better integration with KServe, you can refer to the slides
+[here](https://docs.google.com/presentation/d/1wprxN0n23EMkPRX_PaZZcIzZbn_i8Sh_).
+
+## Training Operator (Trainer) & Katib
```

Comment:
You can also add JAX Support in Training Operator 1.9
https://github.com/kubeflow/trainer/issues/1619